### PR TITLE
feat(Mystera Legacy): add presence

### PR DESCRIPTION
**Pull request by luckycdev**
## Description 

New Presence for [Mystera Legacy](https://mysteralegacy.com)
## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

Image 1

Image 2

</details>

### CODE REVIEW

**Review by RisingSunLight42 — CHANGES_REQUESTED** [review]
logo and thumbnail links are not imgur links. Should be imgur.

**Comment by luckycdev** [comment]
> logo and thumbnail links are not imgur links. Should be imgur.

fixed

File: websites/M/Mystera Legacy/presence.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
+const presence = new Presence({
+		clientId: "1010667786919497789",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey:
+			"https://play-lh.googleusercontent.com/zS3Zi_4PltspasItG2tP2pQujLW6PcR_hGYUy8uMzOEbiWOkS_vlewFcHrn4gVtawCM",
```
Comment (RisingSunLight42) [inline websites/M/Mystera Legacy/presence.ts:9]:
```suggestion
			"https://i.imgur.com/XBaHzfF.png",
```

**Review by RisingSunLight42 — COMMENTED** [review]
Seems a bit low quality for me. There's not dynamic feature used.

**Comment by RisingSunLight42** [comment]
Don't forget to run prettier.

File: websites/M/Mystera Legacy/presence.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+	else if (document.location.pathname.includes("/credits"))
+		presenceData.details = "Viewing the credits";
+	else if (document.location.pathname.includes("/terms"))
+		presenceData.details = "Viewing the Terms of Service";
+	else if (document.location.pathname.includes("/privacy"))
+		presenceData.details = "Viewing the privacy page";
+	else if (document.location.pathname.includes("/forums"))
+		presenceData.details = "Viewing the forums";
+	else if (document.location.pathname.includes("/become-a-supporter"))
+		presenceData.details = "Becoming a Supporter";
```
Comment (DarkVillager) [inline websites/M/Mystera Legacy/presence.ts:49]: maybe use a variable like this and you could use a switch statement

`const { pathname } = document.location`

**Review by DarkVillager — CHANGES_REQUESTED** [review]

File: websites/M/Mystera Legacy/presence.ts [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+	if (presenceData.details) presence.setActivity(presenceData);
+	//Update the presence with no data, therefore clearing it and making the large image the Discord Application icon, and the text the Discord Application name
```
Comment (DarkVillager) [inline websites/M/Mystera Legacy/presence.ts:52]: This comment is unnecesary, it's already clear enough without it.
```suggestion
```

**Review by DarkVillager — CHANGES_REQUESTED** [review]

**Pull request closed without merging**